.github/workflows/triton-benchmarks.yml

```
@@ -112,7 +116,7 @@ jobs:
           python setup.py install
 
       - name: Run Triton Softmax kernel benchmark
-        if: ${{ steps.install.outcome == 'success' && !cancelled() }}
+        if: ${{ steps.install.outcome == 'success' && !cancelled() && !contains(fromJson(inputs.skip_benchmarks), 'fused_softmax.py') }}
```

Note that `inputs` are null when the workflow is triggered by cron. You probably need something like this:
```suggestion
        if: ${{ steps.install.outcome == 'success' && !cancelled() && !contains(fromJson(inputs.skip_benchmarks || '[]'), 'fused_softmax.py') }}
```